esptool/cmds.py

```diff
@@ -737,7 +737,12 @@ def get_key_from_value(dict, val):
             title = "{} extended image header".format(args.chip.upper())
             print(title)
             print("=" * len(title))
-            print("WP pin: {:#02x}".format(image.wp_pin))
+            print(
+                "WP pin:",
+                "disabled"
```

Comment:
Also showing the raw value would be nice in some cases, we can keep it in parentheses.

Reply:
How about:
```
WP pin: 0xee (disabled)
WP pin: 0x12
```
That way the hex value is the same format in both cases.

Reply:
Perfect!